tests/test_geometry.cpp

```diff
@@ -0,0 +1,97 @@
+#include "../src/geometry.h"
+#include <catch.hpp>
+#include <iostream>
+
+using namespace turner;
+
+TEST_CASE("Vector3 default contructor", "[vector3]") {
```

Comment:
Testing `Vector2`, `Vector3`, `Point2`, `Point3` and `Normal3` seems like a low hanging fruit. Should this be a follow up? Should I take this on?